samples/sample_encode/include/pipeline_encode.h

```diff
@@ -168,8 +168,9 @@ struct sInputParams
     mfxU16 IntRefCycleDist;
 
     bool bUncut;
-    bool shouldUseShiftedP010Enc;
-    bool shouldUseShiftedP010VPP;
+    bool shouldUseShifted10BitEnc;
+    bool shouldUseShifted10BitVPP;
+	bool IsSourceMSB;
```

Comment:
wrong alignment

Reply:
alignment is correct - tabs are displayed incorrectly